src/sqlfluff/core/linter/linter.py

```diff
@@ -329,6 +329,10 @@ def remove_templated_errors(
                 result.append(e)
         return result
 
+    @staticmethod
+    def _warn_duplicate_anchors(message: str):  # pragma: no cover
+        linter_logger.warning(message)
```

Comment:
Seems a bit of a pointless function? One line of code to log the param, to save you typing one line of code to log the param? The similar `_warn_unfixable` at least has the error message in the function with only the variable part passed but here the whole message is passed.

Reply:
This function exists primarily in order to let us monkeypatch it at runtime (replacing it with a function that raises an exception).

Reply:
Should we add a comment to that effect?

Reply:
Will do.